pegasus/sites.v3/code.org/public/about/2017.md.erb

```diff
@@ -80,7 +80,7 @@ Code.org by the Numbers
 
 <br>
 
-*URM stands for underrepresented minorities, and includes students who are black / African American, Hispanic/Latino/Latina, Native American/Alaskan, and Native Hawaiian/Pacific Islanders. In the reports for 2014 and 2015, the numbers reported above only counted black and Hispanic students.
+*URM stands for underrepresented minorities, and includes students who are black / African American, Hispanic/Latino/Latina/Latinx, Native American/Alaskan, and Native Hawaiian/Pacific Islanders. In the reports for 2014 and 2015, the numbers reported above only counted black and Hispanic students.
```

Comment:
I know this is unrelated to your change but "Black" should be capitalized here and in the other versions of this page

Reply:
I noticed that too but wasn't sure if I should update it in this PR, but since you mentioned it I'll update it here and on the other Annual Report pages

Reply:
As a general rule, for comments like this, I'm definitely okay with follow-up work here if that makes more sense. Either way, thanks for updating!

Reply:
Ok cool, good to know! 🙏 😄